docker-compose-spark-delta-lake.yml

```
@@ -0,0 +1,50 @@
+version: '3.8'
```

```suggestion
version: '3'
```

elk와 통일시키면 좋을 것 같아요~

---

혹시 elk docker compose 작성 시 3.8이 아닌 3.0을 사용하는 이유가 있을까요?
3.8이 최신으로 보입니다!

---

특별한 이유는 없었던 것 같아요 ㅋㅋ 보통 3을 써서 그랬던 것 같습니다. 혹시 저희가 3.8 버전 compose의 어떤 기능을 사용중인가요?

---

앞서 질문드렸던 것은 궁금증에 여쭤본거구요! 3.8로 가도 상관없습니다~

---

앗 저는 그냥 최신 버전을 선택한거였어서 여쭤봤습니다!ㅎㅎ

---

엇 하은님 이거는 3.8로 가시죠! 최신버전으로 가도 좋을 것 같아요 ㅎㅎ

---

앗 네! 반영했습니다~
[eaa3475](https://github.com/fearless-pioneer/simple-data-flow/pull/4/commits/eaa3475127c0221ba2ca128a516f1e13a18c5464)